src/evm-block-extractor/Dockerfile

```diff
@@ -31,8 +36,16 @@ RUN apt-get update -y \
 
 COPY --from=builder /app/target/release/evm-block-extractor-server /app/evm-block-extractor-server
 
+
+# Copy the entrypoint script into the container
+COPY src/evm-block-extractor/scripts/entrypoint.sh /app/entrypoint.sh
```

Comment:
Why do we need this entrypoint.sh file? it is only a single line of code, this can be directly done in the `cmd` dockerfile section

Reply:
The `CMD` we had initially was not passing the values hence that is why I removed it; tried a lot of combination with the CMD but it really didn't work.

Reply:
I can help you with this issue, I think your problem was that you are using both the entrypoint and the cmd docker instructions that are tricky to use together

Reply:
Fixed this, thanks.